### DISCUSSION TITLE
Feature Request - Support Light theme for web version

### DISCUSSION THREAD

**Discussion by dnet890** [ente-io/ente#1032] [Enhancements]
Hello
I like the web version for this app. But, not everyone like the dark theme. Is it possible to support light theme?

**Comment by andreaferretti** [comment]
Also in the Mac app, which I guess is just a wrapper over some web technology

**Comment by bremensaki** [comment]
Came here from searching to see if I'd missed an option, and can't believe the web UI is dark-only. I signed up on the Android app which was great and followed my system theme preference.

I really struggle with reading text on dark pages, so this is a deal-breaker for me. If there's any change to this on the roadmap anywhere, I'd like to know.

Honestly this should be tagged as an accessibility issue, not a feature request.

**Comment by mnvr** [comment]
This has been done, marking completed.
